src/api/bank-accounts/bank-accounts.md

```diff
@@ -60,6 +60,7 @@ bank transaction can be used to verify a bank account.
 | fullName     | String  | The first and last name of the user.                        |
 | emailAddress | String  | The user's email address.                                   |
 | test         | Boolean | A flag which is present if the Bank Account is for testing. |
+| assetId      | String  | The id of an asset which is associated to the bank account. |
```

Comment:
This could have `Asset` with capitals as it is describing an Asset and then link to the Asset docs perhaps?